lib/traject/dataspace_research_data_config.rb

```diff
@@ -133,6 +133,12 @@
   accumulator.concat DateNormalizer.format_array_for_display(issue_dates)
 end
 
+to_field "issue_date_strict_ssi" do |record, accumulator, _context|
```

Comment:
This is a new field that forces the date into a YYYY-MM-DD format so that we can sort by it.